src/components/Content/Content.tsx

```diff
@@ -26,13 +26,14 @@ const Content: React.FC<TProps> = (props: TProps) => {
         </ul>
       </aside>
       <main className={styles.contentMain}>
-        {props.error && <Errors>{props.error}</Errors>}
-        {props.movies.length === 0 && <Errors>{props.noResults}</Errors>}
+        {props.error && <Errors data-testid="errors">{props.error}</Errors>}
+        {props.movies.length === 0 && <Errors data-testid="errors">{props.noResults}</Errors>}
```

Comment:
Errors componentinde data-testid atadığın için burda bi daha atamana gerek yok. Buraları silebiliriz